client/lxd_operations.go

```diff
@@ -25,10 +30,15 @@ func (r *ProtocolLXD) GetOperationUUIDs() ([]string, error) {
 
 // GetOperations returns a list of Operation struct.
 func (r *ProtocolLXD) GetOperations() ([]api.Operation, error) {
+	err := r.CheckExtension("operations_get_query_all_projects")
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
This check needs to be removed as the function existed before introducing this new extension, and should continue working.